metrics/serve.go

```diff
@@ -70,8 +70,18 @@ func cycleValues(labelKeys []string, labelValues []string, seriesCount int, seri
 	}
 }
 
+func maybeStartTicker(interval int, dontStart bool) (*time.Ticker, error) {
```

Comment:
in both cases if `dontStart` is `true` or `false` the return value for `error` is nil.  Why even return an error if it can only ever be `nil`?